CMakeLists.txt

```
@@ -68,9 +68,9 @@ include (${CARLA_WORKSPACE_PATH}/CMake/CarlaOptions.cmake)
 include (${CARLA_WORKSPACE_PATH}/CMake/Warnings.cmake)
 include (${CARLA_WORKSPACE_PATH}/CMake/Errors.cmake)
 
-if (NOT CMAKE_BUILD_TYPE STREQUAL "Release")
+if (NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
   carla_warning (
-    "CARLA is not set to be built in Release mode. This may cause issues when building CarlaUnrealEditor."
+    "CARLA is set to be built in Debug mode. This may cause issues when building CarlaUnrealEditor."
```

```suggestion
    "CARLA is not set to be built in Debug mode. This may cause issues when building CarlaUnrealEditor."
```

The message is inconsistent with the if check.